Makefile

```diff
@@ -222,10 +222,16 @@ e2e-cleanup:
 
 image-cleanup:
 ifneq (, $(shell which docker))
-# Delete Docker image
+# Delete Docker images
+
 ifneq ($(shell docker images -q $(VELA_CORE_TEST_IMAGE)),)
 	docker rmi -f $(VELA_CORE_TEST_IMAGE)
 endif
+
+ifneq ($(shell docker images -q $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)),)
+	docker rmi -f $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)
+endif
+
 endif
 
```

Comment:
APIServer Image can be added as well.

Reply:
Apiserver image seems not installed as the well like here